masu/external/accounts/db/cur_accounts_db.py

```diff
@@ -49,7 +49,7 @@ def get_accounts_from_source(self):
             auth_credential = provider_accessor.get_authentication()
             billing_source = provider_accessor.get_billing_source()
             customer_name = provider_accessor.get_customer()
-            provider = provider_accessor.get_provider_name()
+            provider = provider_accessor.get_type()
```

Comment:
maybe we should be more explicit with the variable name, `provider_type`?

Reply:
Yeah, I agree.  I'll update this